### ISSUE TITLE
link cannot tap on MacOS

### ISSUE THREAD

**Issue by zhouzhuo810** [fleather-editor/fleather#380]
## Steps to Reproduce

1. clone demo
2. run
3. input www.baidu.com  
4. press `ctrl` + mouse single tap

## Environment
- OS [macOS 14.2.1 (23C71)]
- Flutter version [3.19.6]
- Fleather version [1.16.0]

## Logs

```
GestureRecognizer _getRecognizer(Node segment) {
    print(segment);
    if (_linkRecognizers.containsKey(segment)) {
      return _linkRecognizers[segment]!;
    }

    if (isDesktop || widget.readOnly) {
      _linkRecognizers[segment] = TapGestureRecognizer(
        supportedDevices: {
          PointerDeviceKind.stylus,
          PointerDeviceKind.trackpad,
          PointerDeviceKind.mouse,
          PointerDeviceKind.invertedStylus,
        }
      )
      ..onTapCancel = () {
        print("777");
      }
      ..onSecondaryTap = () {
        print("4444");
      }
      ..onTapUp = (details) {
        print("222");
      }
      ..onTapDown = (details) {
        print("3333");
      }
        ..onTap = () => _tapLink(segment);
      print('111');
    } else {
      _linkRecognizers[segment] = LongPressGestureRecognizer()
        ..onLongPress = () => _longPressLink(segment);
    }
    return _linkRecognizers[segment]!;
  }
```

log :

```
flutter: ⟨www.baiduc.om⟩a
flutter: 111
flutter: ⟨www.baiduc.om⟩a
```

**Comment by zhouzhuo810** [comment]
![image](https://github.com/user-attachments/assets/8e5c0bda-4095-48a2-861a-fc1ffb2e360d)

the first one link can tap

the second link can not tap

**Comment by zhouzhuo810** [comment]
some times can tap , some times can not tap

**Comment by zhouzhuo810** [comment]
and todo some times can not check

**Comment by zhouzhuo810** [comment]
```
[1.14.4](fleather: ^1.14.4)
```
is work well

My code is the newest code

and I changed below:

![image](https://github.com/user-attachments/assets/c47eda15-82fe-485f-b66b-f1c5b3e1d9f4)

because my flutter version is 3.19.6

**Comment by zhouzhuo810** [comment]
It is the same result on Android.

@Amir-P

**Comment by amantoux** [comment]
@zhouzhuo810 this was fixed with #399
